docs/C-Summary/CSS/index.css

```diff
@@ -76,7 +79,8 @@ main{
   padding: 10px;
   display: flex;
   align-items: center;
-  margin: 5px;
+  margin-top: 5px;
```

Comment:
why don't use `margin: 5px auto;` instead define it in two lines?

Reply:
If you use `margin: 5px auto;` it will leave a small 5px gap on the left side of the `div class=block` that you can see thanks to the `h3` Summary on top.

Reply:
And I thought that it might also be better for legibility if I specifically say that it's for `top` and `bottom`. But correct me if I'm wrong.

Reply:
I found out with Martin that `margin: 5px 0;` does work and improve the code, so I will change it in the next commit.